BuildArchive.sh

```diff
@@ -5,8 +5,11 @@
 set -e
 set -x
 
+export GIT_COMMIT=origin/rpm-services-ui
```

Comment:
Will want to delete this line when changes in hoot's rpm-services-ui branch are merged into develop.